src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -221,8 +222,7 @@ export function MessagesList({
 
   // -- Keyboard animation handling
   const {bottom: bottomInset} = useSafeAreaInsets()
-  const nativeBottomBarHeight = isIOS ? 42 : 60
-  const bottomOffset = isWeb ? 0 : bottomInset + nativeBottomBarHeight
+  const bottomOffset = isWeb ? 0 : clamp(60 + bottomInset, 60, 76)
```

Comment:
This 15 pixel range comes from https://github.com/bluesky-social/social-app/blob/03655abb7cfbe8fea3f73fc4a298e54364c1de63/src/view/shell/bottom-bar/BottomBar.tsx#L134